src/shared/libmount-util.h

```diff
@@ -2,19 +2,33 @@
 #pragma once
 
 /* This needs to be after sys/mount.h */
-#include <libmount.h>
+#include <libmount/libmount.h>
 
 #include "macro.h"
 
 DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(struct libmnt_table*, mnt_free_table, NULL);
 DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(struct libmnt_iter*, mnt_free_iter, NULL);
+DEFINE_TRIVIAL_CLEANUP_FUNC_FULL(struct libmnt_cache*, mnt_unref_cache, NULL);
 
 int libmount_parse(
                 const char *path,
                 FILE *source,
                 struct libmnt_table **ret_table,
                 struct libmnt_iter **ret_iter);
 
+/* If /proc/self/mountinfo contains an entry like /dev/root, then libmount will
```

Comment:
Is this really performance sensitive? In the long term `statmount` and `listmount` should be the way to go, with which the whole parsing of /p/s/mountinfo can go.